pkgs/tools/games/gamemode/default.nix

```
@@ -1,45 +1,37 @@
 { lib
 , stdenv
 , fetchFromGitHub
-, fetchpatch
 , libgamemode32
+, makeWrapper
 , meson
 , ninja
 , pkg-config
 , dbus
 , inih
 , systemd
 , appstream
-, makeWrapper
 , findutils
 , gawk
 , procps
+, nix-update-script
 }:
 
-stdenv.mkDerivation rec {
+stdenv.mkDerivation (finalAttrs: {
   pname = "gamemode";
-  version = "1.7";
+  version = "1.8.1";
 
   src = fetchFromGitHub {
     owner = "FeralInteractive";
-    repo = pname;
-    rev = version;
-    sha256 = "sha256-DIFcmWFkoZOklo1keYcCl6n2GJgzWKC8usHFcJmfarU=";
+    repo = "gamemode";
+    rev = "refs/tags/${finalAttrs.version}";
```

Is there any reason not to use references and simplify these?
```suggestion
    repo = finalAttrs.pname;
    rev = finalAttrs.version;
```

---

It's mainly just a convention I've been switching to for the packages I maintain:

- `pname` is not always the same as the repo name, so I've been preferring to decouple anything that used to refer to the derivation's `pname` for consistency. I also find it clearer to read that way.
- Some repos have release tags & branches with the same name, and rev doesn't distinguish between the two when just referring to the name. I've found it better to be safe and explicitly request a tag to avoid accidentally fetching from a branch.